docs/resources/Auto_Moderation.md

```diff
@@ -40,10 +40,17 @@ Rules can be configured to automatically execute actions whenever they trigger.
     {
       "type": 2,
       "metadata": { "channel_id": "123456789123456789" }
+    },
+    {
+      "type": 3,
+      "metadata": {
+        "duration_seconds": 60
+      }
     }
   ],
   "trigger_metadata": {
-    "keyword_filter": ["cat*", "*dog", "*ana*", "i like javascript"]
+    "keyword_filter": ["cat*", "*dog", "*ana*", "i like rust"],
+    "regex_patterns": ["(b|c)at", "^(?:[0-9]{1,3}\.){3}[0-9]{1,3}$"]
```

Comment:
```suggestion
    "regex_patterns": ["(b|c)at", "^(?:[0-9]{1,3}\\.){3}[0-9]{1,3}$"]
```

Reply:
I think it should be fine without the extra escape -- the backslash is there to escape the period. Unless you meant something else i'm not understanding

Reply:
The JSON already uses the \ for string escapes like "\n" for newline, so you need another backslash for regex `"\\."` in json is equivalent to `\.` in normal text.

Reply:
ah JSON, right 👍